src/Components/Artwork/GridItem.tsx

```diff
@@ -44,37 +43,52 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
   private image: HTMLImageElement = null
 
   state = {
-    width: 1,
-    height: 1,
+    isMounted: false,
   }
 
   componentDidMount() {
-    const scale = window.devicePixelRatio
-    const width = this.image.width * scale
-    const height =
-      (this.image.width / this.props.artwork.image.aspect_ratio) * scale
-
     this.setState({
-      width,
-      height,
+      isMounted: true,
     })
   }
 
-  get imageURL() {
+  getImageUrl(breakpoints) {
     const imageURL = this.props.artwork.image.url
-    if (imageURL) {
+
+    if (!imageURL) {
+      return null
+    }
+
+    if (this.state.isMounted) {
+      const [[match]] = Object.entries(breakpoints).filter(([key, val]) => val)
+
+      const getWidth = () => {
+        switch (match) {
+          case "xs":
+          case "sm":
+          case "md":
+            return 400
+          case "lg":
+          case "xl":
+          default:
+            return 300
+        }
+      }
+
+      const scale = window.devicePixelRatio
+      const width = String(getWidth() * scale)
+      const height = String(
+        (this.image.width / this.props.artwork.image.aspect_ratio) * scale
```

Comment:
This shouldn't be `this.image.width` anymore, but the width from above (ie- the 'max' width). We can drop the `innerRef` stuff then as well.

Also, realizing we don't need `String(...)` around this either since we're using string interpolation anyway.

I think we should replace these two lines with:

```
const width = Math.floor(getWidth())
const height = Math.floor(width / this.props.artwork.image.aspect_ratio)
```